spec/models/like_spec.rb

```diff
@@ -0,0 +1,4 @@
+require_relative '../rails_helper'
+RSpec.describe Like, type: :model do
+  # tests go here
+end
```

Comment:
Please be sure to write some tests for `Like` model as it is part of the requirements 🤓